Products/Zuul/tree.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, ob, root=None, parent=None):
         self._parent = parent or None
         self._severity = None
         # allow showing the event severity icons to be configurable
-        if not hasattr(self._root, 'showSeverityIcons'):
```

Comment:
There was a typo that was causing the severity icon lookup to happen for every node instead of just the root.